qutebrowser/browser/commands.py

```diff
@@ -2234,7 +2234,16 @@ def fullscreen(self, leave=False):
             return
 
         window = self._tabbed_browser.widget.window()
-        window.setWindowState(window.windowState() ^ Qt.WindowFullScreen)
+        if window.isFullScreen():
+            window.setWindowState(
+                window.state_before_fullscreen & ~Qt.WindowFullScreen)
+        else:
+            window.state_before_fullscreen = window.windowState()
+            window.setWindowState(
+                window.state_before_fullscreen | Qt.WindowFullScreen)
+        log.misc.debug('state before fullscreen: {}'.format(
+            debug.qflags_key(Qt, window.state_before_fullscreen)))
+
```

Comment:
Pylint is complaining about [too many lines here](https://travis-ci.org/qutebrowser/qutebrowser/jobs/443271586)

Reply:
I removed the superfluous blank line now.